voidcrew/code/game/objects/items/devices/scanners.dm

```diff
@@ -26,3 +26,63 @@
 	var/response = SEND_SIGNAL(M, COMSIG_NANITE_SCAN, user, TRUE)
 	if(!response)
 		to_chat(user, "<span class='info'>No nanites detected in the subject.</span>")
+
+/obj/item/bio_scanner
+	name = "Biological Scanner"
+	desc = "A small device that performs a detailed scan of various living or descesed beings and uploads the collected data to a connected research server."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "forensicnew" // should be temp
+	w_class = WEIGHT_CLASS_SMALL
+	item_state = "electronic" // should be temp
+	lefthand_file = 'icons/mob/inhands/misc/devices_lefthand.dmi' // should be temp
+	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi' // should be temp
+	flags_1 = CONDUCT_1
+	item_flags = NOBLUDGEON
+	slot_flags = ITEM_SLOT_BELT
+	throwforce = 3
+	w_class = WEIGHT_CLASS_TINY
+	throw_speed = 3
+	throw_range = 7
+	var/datum/techweb/stored_research
+	var/value = 0 // the actual reward
+	var/reward = 1500 // base reward before bonus, todo: make this a var in /mob/
+
+/obj/item/bio_scanner/attack(mob/living/M, mob/living/user)
```

Comment:
do not use single letter variable names they should be descriptive and should accurately represent what the variable is storing
this could be:
```
mob/living/attacked_mob
```
or something similar